docs/crd.md

```diff
@@ -38,11 +38,9 @@ Below is the list of fields in the custom resource and their description
     * **taskSlots** `type:int32 required=true`
       Number of task slots per task manager
 
-    * **offHeapMemoryFraction** `type:float64`
+    * **systemMemoryFraction** `type:float64`
       A value between 0 and 1 that represents % of container memory dedicated to system / off heap. The
```

Comment:
I think with the new memory configs (`process.size`) off-heap is actually within the flink portion, rather than part of the system portion (I'm following the diagram [here](https://ci.apache.org/projects/flink/flink-docs-release-1.11/ops/memory/mem_setup.html#configure-total-memory)), so this should just read "% of container memory dedicated to the system."